src/views/dashboard/OnlineEvents.vue

```diff
@@ -0,0 +1,9 @@
+<template>
+  <div>
+    <div class="content">
+
+      <iframe src="https://calendar.google.com/calendar/b/4/embed?height=800&amp;wkst=1&amp;bgcolor=%23ffffff&amp;ctz=Europe%2FBrussels&amp;src=YWVnZWUuZXVfdjVtbjY1MWltZXFwdnM4N3Y0bG43aHIxZjRAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ&amp;color=%237CB342&amp;showCalendars=0&amp;showTabs=1&amp;showPrint=1&amp;showDate=1" style="border:none; height:700px" width="100%" frameborder="0" scrolling="no"></iframe>
```

Comment:
did you check out the ical thing that I've sent? is it working?

Reply:
Ah no I forgot about that one, can you send it again maybe cause I can't find it back now

Reply:
https://stackoverflow.com/questions/30766477/google-calendar-ical-url-for-public-calendar I took a reference from here and just replaced the calendar URL with the one from the iframe